### PR TITLE
Editfunction

### PR DESCRIPTION
**Pull request by J-Massmann**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -45,12 +47,25 @@ export default function App() {
...
   }
 
-  function editRoute(handleData, routes) {
+  function addRoute(handleData, routes) {
     updateDestinations(draft => {
       const destination = draft.find(
         destination => destination.id === handleData.id
       );
       destination.routes = [...destination.routes, routes];
+      console.log(destination);
```
Comment (mwarnck) [inline src/App.js:56]: I think you don´t need this anymore :)

File: src/components/DayCard.js [hunk 2]
```diff
@@ -1,12 +1,15 @@
 import styled from 'styled-components';
 import getDisplayDate from './hooks/getDisplayDate.js';
 
-export default function DayCard({ routeNumber, data }) {
+export default function DayCard({ routeNumber, data, startDate }) {
   const date = getDisplayDate(data.date);
+  const daysDiff = (new Date(data.date) - new Date(startDate)) / 86400000;
```
Comment (mwarnck) [inline src/components/DayCard.js:6]: what is the 8640000 here for?

File: src/pages/DayPlaner.js [hunk 3]
```diff
@@ -44,7 +44,12 @@ export default function Dayplaner({ onGetCurrentDestination }) {
               )})`}
               style={linkStyle}
             >
-              <DayCard key={index} routeNumber={index + 1} data={data} />
+              <DayCard
+                key={index}
```
Comment (mwarnck) [inline src/pages/DayPlaner.js:48]: Isn´t there a better value for the key available?

File: src/pages/EditDay.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+  & h1 {
+    width: 100%;
+    text-align: center;
+    @media (max-width: 230px) {
+      text-align: end;
+    }
+  }
+  img {
+    @media (min-width: 200px) {
+      position: absolute;
```
Comment (mwarnck) [inline src/pages/EditDay.js:73]: only position absolute with no further attributes for positioning like top or right?

**Review by mwarnck — APPROVED** [review]
Looks good to me overall. Just a few little comments from me.

File: src/components/DayCard.js (hunk 2 above)
Comment (J-Massmann) [reply src/components/DayCard.js:6]: this is the amount of milliseconds of one day 😅

**Pull request merged**